### PR TITLE
Marks dashboard chart axis labels for translation

### LINKED ISSUE #10683: Unmarked UI strings on Dashboard graph
##### ISSUE TYPE

 - Bug Report

##### COMPONENT NAME

 - UI

##### SUMMARY

The dashboard graph's axis are not marked.  

image

Strings to mark:
* https://github.com/ansible/awx/blob/devel/awx/ui_next/src/screens/Dashboard/shared/LineChart.jsx#L133
* https://github.com/ansible/awx/blob/devel/awx/ui_next/src/screens/Dashboard/shared/LineChart.jsx#L173

### PR DESCRIPTION
**Pull request by AlexSCorey**
##### SUMMARY
This resolved #10683 and #10778
##### ISSUE TYPE
 - Bugfix Pull Request

##### COMPONENT NAME
 - UI

##### ADDITIONAL INFORMATION
Screen Shot 2021-07-21 at 11 35 04 AM

### CODE REVIEW

**Comment by nixocio** [comment]
@AlexSCorey why do we need to generate the `.po` files again? I have been marking strings to be translated without generating the `.po` files again.  I did think this was not necessary anymore once we upgraded lingui.

**Comment by AlexSCorey** [comment]
This file is a little bit different than the rest of our other ones that need translations but for reasons that aren't 100% clear to me I had to define those translated strings as a variable outside of the function in order for them to be translated.  In order to ling ui to be able to evaluate those strings properly we have to run `npm run extract-strings` to generate new .po files.  We could have forgone that step at this point, and in the future should consider doing so, but I chose not to because this is a translation specific PR/issue. 

We do have automation set up that runs all the necessary commands for us to extract, and compile strings, that we kick off just before we do a release, so I _could_  have just let that handle this work here.  However, for better developer experience, and easier review/testing, I wanted to include the .po files in this PR.

**Review by keithjgrant — APPROVED** [review]

**Comment by tiagodread** [comment]
@AlexSCorey the strings are showing in english even when the browser is in japanese, tried on prod build (make ui-devel) and dev server.

image

File: awx/ui/src/screens/Dashboard/shared/LineChart.js [hunk 1]
```diff
@@ -5,6 +5,8 @@ import { t } from '@lingui/macro';
 import { PageContextConsumer } from '@patternfly/react-core';
 import ChartTooltip from './ChartTooltip';
 
+// const yAxisLabel = t`Job Runs`;
+// const xAxisLabel = t`Date`;
```
Comment (tiagodread) [inline awx/ui/src/screens/Dashboard/shared/LineChart.js]: these comments can be removed?

**Review by tiagodread — APPROVED** [review]
👍🏽

**Pull request merged**